src/BinSkim.Rules/PERules/BA2024.EnableSpectreMitigations.cs

```diff
@@ -241,6 +246,14 @@ public override void AnalyzePortableExecutableAndPdb(BinaryAnalyzerContext conte
                     // mitigations. We do not report here. BA2006 will fire instead.
                     continue;
                 }
+
+                // If the Command Line is not embedded in the binary, then just add a warning for this condition, and there is no point in checking switch states
```

Comment:
>states [](http://example.com/codeflow?start=155&length=6)

you do not have a period at the end of your comment.
your line of code is too long, please break your comment into two lines and columnize it.

Reply:
The line is broken and the period is at the end now.